<?xml version="1.0"?>
<launch>

    <rosparam param="sink_name">"alsa_output.pci-0000_00_1b.0.analog-stereo"</rosparam>
    <rosparam param="meter_rate">10</rosparam>
    <rosparam param="max_sample_value">255</rosparam>
    <rosparam param="display_scale">0</rosparam>

    <node name="vumeter_node" pkg="rospypavumeter" type="vumeter.py" output="screen"/>

</launch>
